recipe/0001-remove-unused-dependency.patch

```diff
@@ -0,0 +1,10 @@
+---	2024-02-23 19:59:42.113969609 +0100
++++	2024-02-23 19:59:44.738215960 +0100
+@@ -22,7 +22,6 @@
+ Requires-Python: >=3.7,<4.0
+ Description-Content-Type: text/x-rst
+ License-File: LICENSE.txt
+-Requires-Dist: jsonpickle

```

Comment:
Why did you remove this? Considering this package is proprietery, I would leave things as is to avoid any unexpected breakage.

Reply:
I feel strongly about this one. I'd prefer if we don't diverge from what upstream says because there is too much unknowns.

Reply:
Ok. So be it.